src/LintingViewExtension/LintingViewExtension.cs

```diff
@@ -55,20 +52,16 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             this.linterManager.PropertyChanged += OnLinterManagerPropertyChange;
         }
 
-        private void OnRequestOpenLinterView(object sender, System.EventArgs e)
+        private void OnViewExtensionOpenRequest(string extensionName)
         {
+            if (extensionName != Name)
+            {
+                return;
+            }
+            
             if (linterMenuItem.IsChecked)
             {
-                if (linterView != null && 
-                    (dynamoView.tabDynamic.SelectedItem as TabItem).Header.ToString() != EXTENSION_NAME)
-                {
-                    dynamoView.tabDynamic.SelectedItem = dynamoView
-                        .ExtensionTabItems
-                        .Where(x => x.Header.ToString() == EXTENSION_NAME)
-                        .FirstOrDefault();
-
-                    linterView.Focus();
-                }
+                this.viewLoadedParamsReference?.AddToExtensionsSideBar(this, this.linterView);
```

Comment:
oh nice! this was exactly what i was looking for, did think this on actually also would switch the tab if it already exist